local-provision-program-intent-engagement.sh

```diff
@@ -0,0 +1,14 @@
+name="program_intent_engagement"
+port="18781"
+
+# Create superuser
+echo -e "${GREEN}Creating super-user for ${name}...${NC}"
+bash -c "echo 'from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser(\"edx\", \"edx@example.com\", \"edx\") if not User.objects.filter(username=\"edx\").exists() else None' | python manage.py shell"
+
+# Provision IDA User in LMS
+echo -e "${GREEN}Provisioning ${name}_worker in LMS...${NC}"
+docker exec -t edx.devstack.lms  bash -c "source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user ${name}_worker ${name}_worker@example.com --staff --superuser"
+
+# Create the DOT applications - one for single sign-on and one for backend service IDA-to-IDA authentication.
+docker exec -t edx.devstack.lms  bash -c "source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker create_dot_application --grant-type authorization-code --skip-authorization --redirect-uris 'http://localhost:${port}/complete/edx-oauth2/' --client-id '${name}-sso-key' --client-secret '${name}-sso-secret' --scopes 'user_id' ${name}-sso ${name}_worker"
+docker exec -t edx.devstack.lms bash -c "source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker create_dot_application --grant-type client-credentials --client-id '${name}-backend-service-key' --client-secret '${name}-backend-service-secret' ${name}-backend-service ${name}_worker"
```

Comment:
This is great! Thanks for adding this to the repo. Could the instructions for how to get the app running (in the README) also be updated?

Reply:
updated!